rest-service/manager_rest/blueprints_manager.py

```diff
@@ -773,6 +825,22 @@ def _verify_deployment_environment_created_successfully(self,
                 'Unexpected deployment status for deployment {0} '
                 '[status={1}]'.format(deployment_id, status))
 
+    # For restore snapshot purpose. To that moment elasticsearch has to be
+    # already restored.
+    def recreate_deployments_enviroments(self):
+        cr_dep_env_tasks = []
+        for dep in self.deployments_list():
+            blueprint = self.get_blueprint(dep.blueprint_id)
+            plan = blueprint.plan
+            deployment_plan = tasks.prepare_deployment_plan(plan,
```

Comment:
refactor `create_deployment_environment` to use explicit fields instead passing deployment plan